roles/matrix-bridge-mautrix-signal/templates/config.yaml.j2

```
@@ -188,14 +188,15 @@ bridge:
     #        * - All Matrix users
     #   domain - All users on that homeserver
     #     mxid - Specific user
-    permissions:
-      '{{ matrix_mautrix_signal_homeserver_domain }}': relay
-      '{{ matrix_mautrix_signal_homeserver_domain }}': user
+    #permissions: 
+    #
+    # Remark: permissions will be set in the defaults/main.yml file of this role 
+    # (see matrix_mautrix_signal_configuration_extension_yaml)
```

```suggestion
    permissions: 
        *: relay
        '{{ matrix_mautrix_signal_homeserver_domain }}': user
```
This way we'd enable the bridge to be used by users on the homeserver and the relay to relay any room member's messages (no matter where they're from - which I think would be an understandable default)